source/precharge/bms/orion.c

```diff
@@ -0,0 +1,12 @@
+#include "orion.h"
+
+#include <stdbool.h>
+#include "can_parse.h"
+#include "common_defs.h"
+
+bool check_errors() {
+    int err = 0;
+    uint16_t relay_state = can_data.orion_info.relay_state;
+    err = (((relay_state & 0x1000) == 0x1000) ? 0 : 1);
```

Comment:
Once the relay state is broken up into its fields this line will become trivial. Rewrite this function without using local variables to combine the stale flag and the error bits.